### PR TITLE
update to 3.25.3

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
- remove patches

- update home, doc, dev urls to https as linter complains

### CODE REVIEW

**Review by skupr-anaconda — APPROVED** [review]
LGTM, but I would wait for approval from other experienced package builders too.

**Review by akabanovs — APPROVED** [review]
LGTM

**Review by danpetry — DISMISSED** [review]

**Comment by danpetry** [comment]
ignore my comment above, all seems fine. Comment was about dropping support for macos sdk <10.5 in cmake 3.19, but we're using cmake v3.22 everywhere else so I guess this isn't an issue.

**Pull request merged**
